prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -176,7 +177,7 @@ func (o *KubernetesOptions) LoadClusterConfigs(callBacks ...func()) (map[string]
 func (o *KubernetesOptions) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&o.kubeconfig, "kubeconfig", "", "Path to .kube/config file. If neither of --kubeconfig and --kubeconfig-dir is provided, use the in-cluster config. All contexts other than the default are used as build clusters.")
 	fs.StringVar(&o.kubeconfigDir, "kubeconfig-dir", "", "Path to the directory containing kubeconfig files. If neither of --kubeconfig and --kubeconfig-dir is provided, use the in-cluster config. All contexts other than the default are used as build clusters.")
-	fs.StringVar(&o.DeckURI, "deck-url", "", "Deck URI for read-only access to the infrastructure cluster.")
```

Comment:
Does the deck uri do anything useful nowadays? It was used in conjunction with a manually-implemented kube client which I believe we don't really use anymore, it does nothing when using the generated or controller-runtime clients

Reply:
Yeah I'd be surprised if it was functional, we can't `LIST` from Deck with the current clients?

Reply:
I will give it a try to remove the flag.